### PR TITLE
Modify createImageBitmap arguments

### PR DESCRIPTION
**Pull request by yangmeijuan**
### Description

The `createImageBitmap` cannot be an Element such as ``,`SVG <image>`,`<video>`,`<canvas>`,but it can get from these Elements.So i modyfied the doc to be less confused.

### Related issues and pull requests

createImageBitmap method lists elements as arguments[#22447](https://github.com/mdn/content/issues/22447)

### CODE REVIEW

File: files/en-us/web/api/createimagebitmap/index.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ createImageBitmap(image, sx, sy, sw, sh, options)
...
-  - : An image source, which can be an {{HTMLElement("img")}}, SVG
-    {{SVGElement("image")}}, {{HTMLElement("video")}}, {{HTMLElement("canvas")}},
-    {{domxref("HTMLImageElement")}}, {{domxref("SVGImageElement")}},
-    {{domxref("HTMLVideoElement")}},
-    {{domxref("HTMLCanvasElement")}}, {{domxref("Blob")}}, {{domxref("ImageData")}},
+  - : An image source, which can be a 
+    {{domxref("HTMLImageElement")}}(or get from {{HTMLElement("img")}}), {{domxref("SVGImageElement")}}(or get from SVG
+    {{SVGElement("image")}}),
+    {{domxref("HTMLVideoElement")}}(or get from {{HTMLElement("video")}}),
+    {{domxref("HTMLCanvasElement")}}(or get from {{HTMLElement("canvas")}}), {{domxref("Blob")}}, {{domxref("ImageData")}},
```
Comment (wbamberg) [inline files/en-us/web/api/createimagebitmap/index.md:39]:
```suggestion

```

File: files/en-us/web/api/createimagebitmap/index.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ createImageBitmap(image, sx, sy, sw, sh, options)
...
     {{domxref("ImageBitmap")}}, or {{domxref("OffscreenCanvas")}} object.
```
Comment (wbamberg) [inline files/en-us/web/api/createimagebitmap/index.md:40]:
```suggestion
  - : An image source, which can any one of the following:
    - {{domxref("HTMLImageElement")}}
    - {{domxref("SVGImageElement")}}
    - {{domxref("HTMLVideoElement")}}
    - {{domxref("HTMLCanvasElement")}}
    - {{domxref("Blob")}}
    - {{domxref("ImageData")}}
    - {{domxref("ImageBitmap")}
    - {{domxref("OffscreenCanvas")})
```

(I think it is better just to list the interfaces here.)

**Review by wbamberg — COMMENTED** [review]
Thanks for your PR, @yangmeijuan ! I think it's better not to include the HTML elements here at all.

File: files/en-us/web/api/createimagebitmap/index.md (hunk 2 above)
Comment (yangmeijuan) [reply files/en-us/web/api/createimagebitmap/index.md:40]: Thanks for your suggestion.But I think the info I put in the bracket can make more sense to readers.
Like ``{{domxref("HTMLImageElement")}}(or get from {{HTMLElement("img")}})``.
What do u think?

Comment (wbamberg) [reply files/en-us/web/api/createimagebitmap/index.md:40]: I think it is better just to list the types that this argument can be. That's just how we write reference docs, we document the types they take, and it's really assumed that people reading the reference map them onto HTML elements, if they have to. For example:
- https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/unpackColorSpace
- https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage#image
- https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Applying_styles_and_colors#patterns

Comment (yangmeijuan) [reply files/en-us/web/api/createimagebitmap/index.md:40]: You are right.Thanks for ur suggestion again.

**Review by wbamberg — APPROVED** [review]
👍 thank you for the fix, @yangmeijuan , and welcome to the MDN community 👋 !

File: files/en-us/web/api/createimagebitmap/index.md [hunk 3]
```diff
@@ -32,12 +32,15 @@ createImageBitmap(image, sx, sy, sw, sh, options)
...
-    {{domxref("HTMLCanvasElement")}}, {{domxref("Blob")}}, {{domxref("ImageData")}},
-    {{domxref("ImageBitmap")}}, or {{domxref("OffscreenCanvas")}} object.
+  - : An image source, which can any one of the following:
+    - {{domxref("HTMLImageElement")}}
+    - {{domxref("SVGImageElement")}}
+    - {{domxref("HTMLVideoElement")}}
+    - {{domxref("HTMLCanvasElement")}}
+    - {{domxref("Blob")}}
+    - {{domxref("ImageData")}}
+    - {{domxref("ImageBitmap")}
```
Comment (wbamberg) [inline files/en-us/web/api/createimagebitmap/index.md:42]:
```suggestion
    - {{domxref("ImageBitmap")}}
```

File: files/en-us/web/api/createimagebitmap/index.md [hunk 4]
```diff
@@ -32,12 +32,15 @@ createImageBitmap(image, sx, sy, sw, sh, options)
...
-    {{domxref("ImageBitmap")}}, or {{domxref("OffscreenCanvas")}} object.
+  - : An image source, which can any one of the following:
+    - {{domxref("HTMLImageElement")}}
+    - {{domxref("SVGImageElement")}}
+    - {{domxref("HTMLVideoElement")}}
+    - {{domxref("HTMLCanvasElement")}}
+    - {{domxref("Blob")}}
+    - {{domxref("ImageData")}}
+    - {{domxref("ImageBitmap")}}
+    - {{domxref("OffscreenCanvas")})
```
Comment (wbamberg) [inline files/en-us/web/api/createimagebitmap/index.md:43]:
```suggestion
    - {{domxref("OffscreenCanvas")}}
```

**Pull request merged**